src/main/java/model/FruitTransaction.java

```diff
@@ -0,0 +1,57 @@
+package model;
+
+public class FruitTransaction {
```

Comment:
Don't begin class or method implementation with empty line. Please remove the empty line at the beginning of the class definition.